src/ImmediateContext.cpp

```diff
@@ -4178,7 +4178,11 @@ void TRANSLATION_API ImmediateContext::DeleteRenameCookie(Resource* pRenameResou
     auto iter = std::find_if(LockedContainer->begin(), LockedContainer->end(),
                              [pRenameResource](unique_comptr<Resource> const& r) { return r.get() == pRenameResource; });
     assert(iter != LockedContainer->end());
-    pRenameResource->UsedInCommandList(COMMAND_LIST_TYPE::GRAPHICS, GetCommandListID(COMMAND_LIST_TYPE::GRAPHICS));
+

```

Comment:
I think it's worth a comment explaining that decode input buffers are a special case for map discard, otherwise it should be a graphics op.

Reply:
Done.